pkgs/by-name/ru/runpodctl/package.nix

```diff
@@ -0,0 +1,32 @@
+{ lib
+, buildGoModule
+, fetchFromGitHub
+}:
+
+buildGoModule rec {
+  pname = "runpodctl";
+  version = "1.14.2";
+
+  src = fetchFromGitHub {
+    owner = "runpod";
+    repo = pname;
+    rev = "v${version}";
+    hash = "sha256-eemg8W8kTjT9k3TKi+pqRisoxD+7KjetZoWFWbpHFV8=";
+  };
+
+  vendorHash = "sha256-9UkHYeDTZlEKEBfazj7N7QJrWcvVi5StxtYE70OF8tw=";
+
+  postInstall = ''
+    rm $out/bin/docs # remove the docs binary
+    mv $out/bin/cli $out/bin/runpodctl
+  '';
+
+  meta = with lib; {
+    homepage = "https://github.com/runpod/runpodctl/";
+    description = "CLI tool to automate / manage GPU pods for runpod.io.";
+    changelog = "https://github.com/runpod/runpodctl/raw/v${version}/CHANGELOG.md";
+    license = licenses.mit;
```

Comment:
https://github.com/runpod/runpodctl/blob/main/LICENSE

Is it also distributed with MIT license?

Reply:
sorry, fixed